scripts/test-simple.ts

```diff
@@ -15,6 +15,17 @@ async function main(model?: string) {
     max_tokens: 1024,
     stop: ['</s>'],
   });
+
+  // Unauthorized requests should fail with a 401
+  let gotExpectedError = false;
+  try {
+      await completion(prompt, {model, apiKey: "BADKEY"});
+  } catch (e: any) {
+    gotExpectedError = e.message == "Status: 401";
+  }
+  if (!gotExpectedError) {
+    throw new Error("Unauthorized request returned unexpected response")
+  }
```

Comment:
nit: we can just re-throw the error in the try if not 401, no need to do it separately IMO. Also, we should either use `startsWith` or strict equal.